### DISCUSSION TITLE
Using this to download anime

### DISCUSSION THREAD

**Discussion by Jurrein** [Serene-Arc/bulk-downloader-for-reddit#635] [Q&A] [answered]
Would it be possible to tweak bdfr so that it can download anime from sites? For example from https://animixplay.to/ , if you look on the site you can see the layout for most generic anime sites.

**Comment by Serene-Arc** [comment, marked as answer]
For that, the BDFR wouldn't be useful, nor would it fit the purpose of the tool. For videos, the BDFR mostly piggybacks off of [the yt-dlp project](https://github.com/yt-dlp/yt-dlp). This is the tool you want, it supports a large number of sites. No guarantees as to whether it supports that one, though you could make a request.
